vtest/gen-ref.bat

```diff
@@ -1,7 +1,7 @@
 
 rem create reference
 
-set MSCORE=..\msvc.install_x64\bin\musescore.exe
+set MSCORE=..\msvc.install_x64\bin\musescore4.exe
 set DPI=130
 
 %MSCORE% %1.mscz -r %DPI% -o %1.png
```

Comment:
Also here it needs to be `%MSCORE% %1.mscx -r %DPI% -o %1.png`, mscx vs. mscz, as I just found out